tests/e2e/specs/api-cache.test.js

```diff
@@ -28,15 +28,29 @@ import {
 	resetSiteKit,
 	safeLoginUser,
 	setupSiteKit,
-	testSiteNotification,
 	useRequestInterception,
-	wpApiFetch,
 } from '../utils';
 import { deleteAuthCookie } from '../utils/delete-auth-cookie';
 
-const goToSiteKitDashboard = async () => {
-	await visitAdminPage( 'admin.php', 'page=googlesitekit-dashboard' );
-};
+async function goToWordPressDashboard() {
+	await visitAdminPage( 'index.php' );
+}
+
+async function googleSiteKitAPIgetTime( options ) {
+	await page.waitForFunction(
+		() =>
+			window.googlesitekit !== undefined &&
+			window.googlesitekit.api !== undefined
+	);
+
+	return await page.evaluate(
+		( d, o ) => {
+			return window.googlesitekit.api.get( 'e2e', 'util', 'time', d, o );
+		},
+		null, // data/params
```

Comment:
Would you mind expanding this comment a bit for clarity?

Reply:
Turns out I didn't end up using them as I thought I might so I've removed the parameters and clarified the comments.